.circleci/config.yml

```diff
@@ -32,8 +32,6 @@ workflows:
           requires:
             - go-build
           filters:
-            branches:
-              only: master
```

Comment:
Hit this testing on giraffe. We should always push to aliyun to make testing easier.